dll/win32/msi/msvchelper.h

```diff
@@ -1,6 +1,6 @@
 
 #ifdef __i386__
-#define __ASM_DEFINE_FUNC(name,suffix,code)
+#define __ASM_DEFINE_FUNC(name,code)
```

Comment:
There's no need to redefine it at all, since this is how it's already defined in `wine/asm.h`. So please just delete this line.

Reply:
Well, with regard to fixing the warnings only, an additional sync change would rather be to add `#if !defined(_MSC_VER)`...
Nonetheless, if you confirm the local (re)definition is unwanted, because the full global definition should always apply, then I would concur to remove it.

Reply:
I have no idea what you're talking about. This file is entirely ROS-specific, and is used only in MSVC builds. The global definition matches your new definition exactly, therefore this line is pointless and should be deleted.

Reply:
Right, obviously.
Fully removed.